src/Void/Init.fs

```diff
@@ -49,6 +62,9 @@ module Init =
                                       InsertModeInputHandler(),
                                       setInputMode inputModeChanger bus.publish)
         modeService.subscribe bus
+        if options.enableVerboseMessageLogging 
+        then MessageLog.Service.subscribe bus
+        else ()

```

Comment:
F# tidbit: because your `if...then...` evaluates to unit, you don't need the else clause.